featurebyte/query_graph/sql/aggregator/forward.py

```diff
@@ -63,13 +64,9 @@ def _get_aggregation_subquery(self, specs: list[ForwardAggregateSpec]) -> LeftJo
         horizon_in_seconds = 0
         if spec.parameters.horizon:
             horizon_in_seconds = pd.Timedelta(spec.parameters.horizon).total_seconds()
-        end_point_expr_current: Expression = cast(
-            Expression,
-            parse_one(f"{point_in_time_epoch_expr.sql()}"),
-        )
-        end_point_expr: Expression = cast(
-            Expression,
-            parse_one(f"{point_in_time_epoch_expr.sql()} + {horizon_in_seconds}"),
+        end_point_expr_current: Expression = cast(Expression, point_in_time_epoch_expr)
```

Comment:
nit: I suspect the cast and explicit type annotation is unnecessary.

Reply:
haha yes, forgot to rm.